build/module-builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:12.18.1-alpine as builder
+FROM node:12.22.10-alpine as builder
```

Comment:
Since the repository uses and requires NodeJS 12.18.1, I think it would be best to stick with this version in our docker images too. This would prevent any weird issues we can get by not using the same version when developing and deploying.

Reply:
I don't think it is something we will be doing soon. Since the version of `pkg` we use controls which version of NodeJS we are using and considering that we have had many issues with updating the library, I think we will keep using this version as we want binaries.

Any reason why you would prefer this version?

Reply:
Many packages are only supporting node 18.22+, that is the reason, is there anything that I can do to help with that?

Reply:
@laurentlp one example is the one below:

error graphql-executor@0.0.18: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.16.0 || >=16.0.0". Got "12.18.1"
#25 26.85 error Found incompatible module. 

I'm looking into why this is the case

Reply:
@laurentlp we may want to consider that support for NodeJS 12 is ending, so maybe it's a good time to look at this, let me know in case I can help with anything regarding that, otherwise, please let me know if we just remove this upgrade from the PR